### PR TITLE
gh-97956: Mention `generate_global_objects.py` in `AC How-To`

### PR DESCRIPTION
**Pull request by sobolevn**
* Issue: gh-97956

### CODE REVIEW

**Comment by sobolevn** [comment]
Looks like GitHub Actions are broken right now. I will re-run the CI a bit later.

**Comment by sobolevn** [comment]
Re-triggering

**Review by blockchainGuru1018 — APPROVED** [review]

File: Doc/howto/clinic.rst [hunk 1]
```diff
@@ -539,7 +539,17 @@ Let's dive in!
         };
 
 
-16. Compile, then run the relevant portions of the regression-test suite.
+16. Code generated by Argument Clinic might use ``_Py_ID`` macro inside.
```
Comment (larryhastings) [inline Doc/howto/clinic.rst]: 16. Argument Clinic may generate new instances of ``_Py_ID``.  For example::

File: Doc/howto/clinic.rst [hunk 2]
```diff
@@ -539,7 +539,17 @@ Let's dive in!
...
+
+    Sample::
+
+        &_Py_ID(new_unique_py_id)
+
+    It means that you would need to run ``Tools/scripts/generate_global_objects.py``
```
Comment (larryhastings) [inline Doc/howto/clinic.rst]: If it does, you'll have to run ``Tools/scripts/generate_global_objects.py`` to regenerate the list of precompiled identifiers at this point.

**Review by larryhastings — COMMENTED** [review]
It's a good idea to add this step to the documentation.  But I have some suggestions on how to rewrite the text to make it a little clearer.  Please take a look!

File: Doc/howto/clinic.rst (hunk 2 above)
Comment (sobolevn) [reply Doc/howto/clinic.rst]: Thanks, done!

It is always great to have native-speakers to double check things like this. I think in another language and it does not translate that well into English in some cases :)

**Comment by sobolevn** [comment]
Looks like the CI failure is somehow related to https://github.com/python/cpython/commit/f612565bd32d4ab0945798da775eea070f08b6fe

**Comment by JulienPalard** [comment]
@sobolevn I think that the CI issues has been fixed (sphinx-lint side for some false postives, and cpython side for the few true positives), can you please rebase your branch on top of main?

**Comment by sobolevn** [comment]
It works now 🎉

**Comment by JulienPalard** [comment]
> It works now tada

Yeah! It's better like that :)

Sorry for the inconvenience :(

**Pull request merged**